doc/user/USAGE.md

```diff
@@ -54,3 +54,7 @@ Here is the list of plugins provided officially by F3D:
 - **occt**: STEP and IGES support
 
 > Note: If you downloaded the binaries from the Release page, it's not necessary to specify manually the plugins above, all of them are loaded automatically.
+
+You can also try plugins maintained by the community. If you have created a plugin and want it in the following list, please submit a pull request.
```

Comment:
If you have created a plugin and would like it to be listed here, please submit a pull request.